### ISSUE TITLE
Vault UI - Sometime got a blank screen when using OIDC authentication

### ISSUE THREAD

**Issue by Zenithar** [hashicorp/vault#8369]
# Description

We are using Vault 1.3.2 with OIDC authentication via Okta IDP.

**To Reproduce**

* Setup a vault cluster with OIDC via Okta
* Login with UI
* Get out from UI
* Do some work 
* Get back to UI
  * Sometimes it says your "you need to reauthenticate" (expected)
  * Sometimes the UI is stuck to a white page with JS errors in the console 

JS ERROR :
```
Error while processing route: vault.index n.policies is null checkForRootToken/<@https://vault.domain.tld/ui/assets/vault-14dca2e4637df4256c70afe78ca6c71d.js:2244:1
```

**Expected behavior**

Be able to view root secrets or ask for reauthentication.

**Environment:**

* Vault Server Version: 1.3.2
* Vault CLI Version: 1.3.2
* Vault UI Browser : Firefox / Chrome
* Server Operating System/Architecture: Ubuntu x86_64

**Additional context**

A workaround consists in clearing the localStorage and re-authenticate to be able to use Vault UI as expected.

**Comment by grantslape** [comment]
Possibly related to #7349 ?  We experience the same issue using userpass on token renewal.

**Comment by djvs** [comment]
```n&&n.policies.includes("root")&&e.removeTokenData(t)})},authenticate:function(){var e,t=(e=regeneratorRuntime.mark(function e(){var t,n,a,s,r,l=arguments```

n.policies is null here.  Pretty annoying bug.

**Comment by hashishaw** [comment]
Apologies for the late response -- this was fixed as of #9467!
